lib/spack/spack/environment/environment.py

```diff
@@ -1016,7 +1048,9 @@ def env_file_config_scope(self):
 
     def config_scopes(self):
         """A list of all configuration scopes for this environment."""
-        return self.included_config_scopes() + [self.env_file_config_scope()]
+        return check_disallowed_env_config_mods(
+            self.included_config_scopes() + [self.env_file_config_scope()]
```

Comment:
This is taking in more scopes than I thought when I use it. The `site` scope seems to be coming into play and not just the `spack.yaml` and `includes` scopes.  Is there a way for me to filter the scopes?  @becker33 any ideas on what to do here?

Reply:
NVM It was due to the `spack.config.override`.